packages/compartment-mapper/src/import-archive.js

```diff
@@ -291,14 +346,26 @@ export const parseArchive = async (
 
   /** @type {ExecuteFn} */
   const execute = async options => {
-    const { globals, modules, transforms, __shimTransforms__, Compartment } =
-      options || {};
+    const {
+      globals,
+      modules,
+      transforms,
+      __shimTransforms__,
+      Compartment,
+      importHook: exitModuleImportHook,
+    } = options || {};
+
+    const internalExitModuleImportHook = exitModuleImportHookMaker({
```

Comment:
`internal` is less specific than naming the boundary we’re inside, which I believe is `compartment`.